internal/pkg/cli/deploy/svc.go

```diff
@@ -969,26 +976,36 @@ func (d *workloadDeployer) pushEnvFileToS3Bucket(in *pushEnvFilesToS3BucketInput
 	return envFileARN, nil
 }
 
-type pushAddonsTemplateToS3BucketInput struct {
-	templater templater
-	uploader  uploader
-}
+func (d *workloadDeployer) pushAddonsTemplateToS3Bucket() (string, error) {
+	if d.addons == nil {
+		return "", nil
+	}
 
-func (d *workloadDeployer) pushAddonsTemplateToS3Bucket(in *pushAddonsTemplateToS3BucketInput) (string, error) {
-	tmpl, err := in.templater.Template()
-	if err != nil {
-		var notFoundErr *addon.ErrAddonsNotFound
-		if errors.As(err, &notFoundErr) {
-			// addons doesn't exist for service, the url is empty.
-			return "", nil
+	if false { // TODO remove to enable packaging addons
```

Comment:
lol 😂 should we remove it now? Isn't this the last milestone

Reply:
no 😤 this is a _refactor_ pr. i'm very excited for my `feat` pr